feat(insights): Updates webvital docs to reflect new changes with performance score calculation (#10656)
getsentry/sentry-docs: docs/product/insights/web-vitals/index.mdx

**Performance Score** is a rating from 0 to 100 that summarizes the perceived performance of your web application. Page loads and Interactions on your web application are scored based on their **Web Vitals**. Each **Web Vital** is given a rating from 0 to 100 through a [Log-Normal Distribution](https://www.desmos.com/calculator/o98tbeyt1t), which is then weighed and aggregated by page and overall for your entire web application.

> **Page Loads** contribute up to 4 individual **Web Vital** components found on each event. Interactions contribute to **INP** (Interaction to Next Paint). The default weights and score rating thresholds for each **Web Vital** can be found in the tables below.

Reviewer: ```suggestion
**Page Loads** can have up to 4 individual **Web Vital** components that can be found on each event, while interactions contribute to **INP** (Interaction to Next Paint). The default weight and score-rating thresholds for each **Web Vital** can be found in the tables below.
```

| [Cumulative Layout Shift](/product/insights/web-vitals/web-vitals-concepts/#cumulative-layout-shift-cls) (CLS)   | [0.1](https://www.desmos.com/calculator/irdoqfftdf)    | [0.25](https://www.desmos.com/calculator/irdoqfftdf)   | 15%   |
| [First Contentful Paint](/product/insights/web-vitals/web-vitals-concepts/#first-contentful-paint-fcp) (FCP)     | [1800ms](https://www.desmos.com/calculator/ezkvtlqxkf)  | [3000ms](https://www.desmos.com/calculator/ezkvtlqxkf) | 15%   |
| [Time To First Byte](/product/insights/web-vitals/web-vitals-concepts/#time-to-first-byte-ttfb) (TTFB)           | [800ms](https://www.desmos.com/calculator/xcpjhfvwfo)  | [1800ms](https://www.desmos.com/calculator/xcpjhfvwfo)  | 10%   |
| [Interaction to Next Paint](/product/insights/web-vitals/web-vitals-concepts/#interaction-to-next-paint-inp) (INP)   | [200ms](https://www.desmos.com/calculator/jaqxodpx8d)  | [500ms](https://www.desmos.com/calculator/jaqxodpx8d)  | 30%   |

<Note>
These thresholds are meant to match Google Lighthouse as closely as possible, but there may be some differences because Sentry operates on real user data, whereas Google Lighthouse operates on lab data.

> Some browsers may not support all **Web Vitals** used in Sentry's **Performance Score** calculation. Find out which Web Vitals are supported by your browser in the [Browser Support](/product/insights/web-vitals/web-vitals-concepts/#browser-support) table.

Reviewer: ```suggestion
Some browsers may not support all **Web Vitals** used in Sentry's **Performance Score** calculation. Look at the [Browser Support](/product/insights/web-vitals/web-vitals-concepts/#browser-support) table to see which Web Vitals are supported by your browser.
```
